payments/forms.py

```diff
@@ -51,8 +51,8 @@ class CreditCardPaymentForm(PaymentForm):
             ' For American Express the four digits found on the front side.'))
 
     def __init__(self, *args, **kwargs):
-        super(CreditCardPaymentForm, self).__init__(
-            hidden_inputs=False, *args,  **kwargs)
+        kwargs['hidden_inputs'] = False
```

Comment:
You need to check if `kwargs` does not already contain `hidden_inputs` or you may end up silently discarding an unsupported value (`CreditCardPaymentForm(hidden_inputs=True)` should fail).